src/Backend/Executor.ts

```diff
@@ -55,7 +55,7 @@ class ExecutorBase implements Executor {
   }
 
   require(): DeviceSpec {
-    throw new Error('Invalid require Spec call.');
+    throw new Error('Invalid DeviceSpec call.');
```

Comment:
`DeviceSpec` is a return type and I am not sure whether `'Invalid DeviceSpec call.'` is appropriate.

For example, if we have a method like the following, 
`'invalid int call` sounds not appropriate.

```
int add(a, b) {  // return int type
   if (some error situation)
      throw new Error('invalid int call')
}
```

Reply:
um.... I will fix it.